src/api/app/views/webui/user/_involvement.html.haml

```diff
@@ -1,31 +1,6 @@
 .card.mb-3#involvement
-  - if feature_enabled?(:responsive_ux)
-    = render partial: 'webui/user/responsive_ux/tabs_profile', locals: { involved_packages: involved_packages,
-                                                                         involved_projects: involved_projects, owned: owned }
-  - else
-    .bg-light
-      %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ role: 'tablist' }
-        %li.nav-item
-          = link_to('#involved-packages', id: 'involved-packages-tab', class: 'nav-link active text-nowrap',
-                    data: { toggle: 'tab' }, role: 'tab', aria: { controls: 'involved-packages', selected: true }) do
-            Involved Packages
-            %span.badge.badge-primary
-              = involved_packages.size
-        %li.nav-item
-          = link_to('#involved-projects', id: 'involved-projects-tab', class: 'nav-link text-nowrap',
-                    data: { toggle: 'tab' }, role: 'tab', aria: { controls: 'involved-projects', selected: false }) do
-            Involved Projects
-            %span.badge.badge-primary
-              = involved_projects.size
-        %li.nav-item
-          = link_to('#owned', id: 'owned-tab', class: 'nav-link text-nowrap',
-                    data: { toggle: 'tab' }, role: 'tab', aria: { controls: 'owned', selected: false }) do
-            Owned Projects/Packages
-            %span.badge.badge-primary
-              = owned.size
-        %li.nav-item.dropdown
-          %a.nav-link.dropdown-toggle{ href: '#', 'data-toggle': 'dropdown', role: 'button', 'aria-expanded': 'false', 'aria-haspopup': 'true' }
-          .dropdown-menu.dropdown-menu-right
+  = render partial: 'webui/user/tabs_profile', locals: { involved_packages: involved_packages,
```

Comment:
As we discussed:

```suggestion
  = render partial: 'tabs_profile', locals: { involved_packages: involved_packages,
```

Reply:
In this exact case, we need to specify the path because, by convention, it infers the plural: ~webui/users~.

```
 Showing /obs/src/api/app/views/webui/user/_involvement.html.haml where line #2 raised:

 Missing partial webui/users/_tabs_profile,
```

Reply:
Oh, alright then!